custom/icds_reports/templates/icds_reports/icds_app/mobile/busy.html

```diff
@@ -0,0 +1,5 @@
+{% load staticfiles %}
+<div class="position-fixed w-100 h-100vh d-flex justify-content-center align-items-center"
+     style="background: rgba(0,0,0,0.5); top: 0; z-index: 99;">
+    <img src="{% static "icds_reports/mobile/images/loader.gif" %}" style="width: 125px; height: 125px">
```

Comment:
For consistency with HQ, can you switch this to `<i class='fa fa-spin fa-spinner'></i>` and use font-size to resize it? We don't use loading images anymore - there's no harm in using them, this is purely a consistency comment. I also don't see the loader.gif file in the codebase.

Reply:
I swapped this in (and added the loader as well). I suspect we might get push back from product team but I'd rather do it this way too so let's see what they say.